src/templates/css/styles.css

```diff
@@ -5,6 +5,7 @@ figure {
 /* layout */
 .body {
   background: #fff;
+  font-family: Roboto,sans-serif;
```

Comment:
We can continue following Material template you are following. I'll revert my change.